install_config/install/docker_registry.adoc

```diff
@@ -11,7 +11,7 @@
 
 toc::[]
 
-ifdef::openshift-origin,openshift-origin,openshift-enterprise,openshift-dedicated[]
+ifdef::openshift-origin,openshift-online,openshift-enterprise,openshift-dedicated[]
```

Comment:
Correct that this was a typo here, but can probably just drop online from the ifdef list since the install&config book doesn't show up for Online anyway.